vhdbuilder/packer/install-dependencies.sh

```diff
@@ -258,11 +258,17 @@ ls -ltr /opt/gpu/* >> ${VHD_LOGS_FILEPATH}
 installBpftrace
 echo "  - $(bpftrace --version)" >> ${VHD_LOGS_FILEPATH}
 
-installBcc
-cat << EOF >> ${VHD_LOGS_FILEPATH}
-  - bcc-tools
-  - libbcc-examples
-EOF
+PARENT_DIR=$(pwd)
+
```

Comment:
nit: add a comment about creating a subshell to install in the background? This is not a common pattern in Agentbaker so it'll be good to note for the future